crates/viewer/re_space_view_spatial/src/visualizers/boxes3d.rs

```diff
@@ -43,133 +35,37 @@ impl Default for Boxes3DVisualizer {
 // NOTE: Do not put profile scopes in these methods. They are called for all entities and all
 // timestamps within a time range -- it's _a lot_.
 impl Boxes3DVisualizer {
-    #[allow(clippy::too_many_arguments)]
     fn process_data<'a>(
-        &mut self,
-        ctx: &QueryContext<'_>,
-        line_builder: &mut LineDrawableBuilder<'_>,
-        mesh_instances: &mut Vec<MeshInstance>,
-        query: &ViewQuery<'_>,
+        pmvb: &mut ProcMeshVisBuffer<'_, Fallback>,
```

Comment:
nit: `pmvb` isn't exactly an intuitive abbreviation. `proc_mesh_buffer` maybe? (or in line with the rename suggestion on the type itself `proc_mesh_drawables`)

Reply:
I made it `builder`. I hope that's okay.